libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -870,7 +870,7 @@ string IRGenerator::dispatchRoutine(ContractDefinition const& _contract)
 				<callValueCheck>
 				<?+params>let <params> := </+params> <abiDecode>(4, calldatasize())
 				<?+retParams>let <retParams> := </+retParams> <function>(<params>)
-				let memPos := <allocate>(0)
+				let memPos := <allocate>()
```

Comment:
Funny, by the way, that it didn't actually allocate the proper size anyways, but always just zero :-).